README.md

```diff
@@ -354,6 +354,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 
 ### Linter
 
+* [dockfmt](https://github.com/jessfraz/dockfmt) 🤕 - Dockerfile formatter and parser by [@jessfraz][jessfraz]
```

Comment:
:nerd_face: symbol for CLI tools is missing, too.

Reply:
Is this not a little redundant?  I would assume a CLI tag is for CLI tools for things that are normally not CLI e.g. reg as CLI client to docker hub?

Reply:
In my opinion, we really have to re-think which tag is used for what purpose, except the :heavy_dollar_sign: and :face_with_head_bandage: icons. And we should clearly mention it in the contribution guidelines. Otherwise, it's going to be confusing and misleading.

Reply:
According to PR #447 , no need to put a CLI symbol tag here.